test/test.ts

```diff
@@ -60,6 +60,48 @@ describe('vue-class-component', () => {
     expect(c.b).to.equal(2)
   })
 
+  it('data: should collect from decorated class properties', () => {
+
+    const decorator1 = (value: any) => (_: any, __:any ): any => {
+      return {
+        enumerable: true,
+        value
+      }
+    }
+
+    const decorator2 = (value: any) => (_: any, __:any ): any => {
```

Comment:
`(_: any, __:any ): any`

Please fix the position of space. (add a space after second colon and remove the space before closing brace)
Looks like there is a kind of things in other place.